specs/AppLifecycle/EnvironmentVariables/AppLifecycle - Environment Variables (EV).md

```diff
@@ -281,20 +282,11 @@ Setting a **User** or **System** variable will persist that variable beyond the
 life of the process in the user profile or system profile.
 
 Unpackaged apps can use the existing native Win32 Get/SetEnvironment and
-registry APIs. They can also use the managed System.Environment class, and
-specifically the methods GetEnvironmentVariable, GetEnvironmentVariables and
-SetEnvironmentVariable. We are introducing the new API so that non-fulltrust
-packaged apps can access some of the same functionality. Key existing behaviors
-are listed in the table below:
-
-| App Type       | API Type | Process Vars | User Vars   | Machine Vars |
-| -------------- | -------- | ------------ | ----------- | ------------ |
-| Unpackaged     | Native   | Read, Write  | Read, Write | Read, Write  |
-| Unpackaged     | Managed  | Read, Write  | Read, Write | Read, Write  |
-| Desktop Bridge | Native   | Read, Write  | Read, Write | Read, Write  |
-| Desktop Bridge | Managed  | Read, Write  | Read, Write | Read, Write  |
-| UWP            | Native   | Read, Write  | Read        | Read         |
-| UWP            | Managed  | Read, Write  | Read        | (none)       |
+registry APIs. Unpackaged .NET apps can use [System.Environment](https://docs.microsoft.com/en-us/dotnet/api/system.environment?view=net-6.0) class, and
+specifically the methods [GetEnvironmentVariable](https://docs.microsoft.com/en-us/dotnet/api/system.environment.getenvironmentvariable?view=net-6.0), 
+[GetEnvironmentVariables](https://docs.microsoft.com/en-us/dotnet/api/system.environment.getenvironmentvariables?view=net-6.0)
+and [SetEnvironmentVariable](https://docs.microsoft.com/en-us/dotnet/api/system.environment.setenvironmentvariable?view=net-6.0).
+We are introducing the new API so non-fulltrust apps can access the same functionality.
```

Comment:
Not sure why you call out "non-fulltrust" apps. Any app (Win32, UWP, full-trust MSIX, etc.) can use either the native or .NET APIs. They might not work given access controls (e.g. UWP can't write HKCU; non-elevated can't write HKLM) but this new API doesn't change that.

Reply:
I see.  This was ahold over from Project Reunion with UWP support.  EM allowed UWP apps to read EV's at the user and machine scope.

But, no more UWP.  I'll take this out.